themes/vue/layout/partners-page.ejs

```diff
@@ -0,0 +1,78 @@
+<div id="partners">
+  <p>
+    Vue Partners are premium shops that provide first-class Vue consulting and development.
+  </p>
+
+  <h2>Active Partners</h2>
+
+  <% for (let partner of page.partners_list) { %>
+    <div class="partner">
+      <div class="logo">
+        <img src="<%- url_for(`/images/${partner.logo}`) %>" alt="<%= partner.name %>">
```

Comment:
Would it be possible to use a remote logo? E.g. something on Cloudinary?

Reply:
Note: using an external image may help track statistics.

Reply:
Yeah we can make that work.